src/EventStore.Core/Util/Opts.cs

```diff
@@ -218,6 +218,9 @@ public static class Opts {
 
 		public const string CertificateSubjectNameDescr = "The certificate subject name.";
 		public static readonly string CertificateSubjectNameDefault = string.Empty;
+		
+		public const string ClientCertificateCommonNameDescr = "The certificate subject name.";
```

Comment:
@pgermishuys I think that the name may be a bit confusing because may support client certificate authentication from external clients later on.

what do you think about `CertificateReservedNodeCommonName` or `CertificateReservedNodeCN` ? with a description of: `The reserved common name to authenticate EventStoreDB nodes/servers from certificates`

Reply:
That's fair, i'll rename the option to `CertificateReservedNodeCommonName`

Reply:
thanks @pgermishuys